__tests__/PostcardsApi.spec.ts

```diff
@@ -99,51 +99,85 @@ describe("postcardsApi", () => {
     let previousUrl = "";
     let postcardList: Postcard[] = [];
     beforeAll(async () => {
-      // ensure there are at least 3 cards present, to test pagination
-      const postcard1 = new PostcardEditable({
-        to: ADDRESSES_EDITABLE[1],
-        from: ADDRESSES_EDITABLE[0],
-        front: FILE_LOCATION_4X6,
-        back: FILE_LOCATION_4X6,
-        use_type: "operational",
-      });
-      const postcard2 = new PostcardEditable({
-        to: ADDRESSES_EDITABLE[3],
-        from: ADDRESSES_EDITABLE[0],
-        front: FILE_LOCATION_4X6,
-        back: FILE_LOCATION_4X6,
-        use_type: "operational",
-      });
-      const postcard3 = new PostcardEditable({
-        to: ADDRESSES_EDITABLE[6],
-        from: ADDRESSES_EDITABLE[1],
-        front: FILE_LOCATION_4X6,
-        back: FILE_LOCATION_4X6,
-        use_type: "operational",
-      });
-      await postcardsApi.create(postcard1);
-      await postcardsApi.create(postcard2);
-      await postcardsApi.create(postcard3);
+      // Create enough postcards to ensure pagination works
+      const postcardsToCreate = [
+        new PostcardEditable({
+          to: ADDRESSES_EDITABLE[1],
+          from: ADDRESSES_EDITABLE[0],
+          front: FILE_LOCATION_4X6,
+          back: FILE_LOCATION_4X6,
+          use_type: "operational",
+        }),
+        new PostcardEditable({
+          to: ADDRESSES_EDITABLE[3],
+          from: ADDRESSES_EDITABLE[0],
+          front: FILE_LOCATION_4X6,
+          back: FILE_LOCATION_4X6,
+          use_type: "operational",
+        }),
+        new PostcardEditable({
+          to: ADDRESSES_EDITABLE[6],
+          from: ADDRESSES_EDITABLE[1],
+          front: FILE_LOCATION_4X6,
+          back: FILE_LOCATION_4X6,
+          use_type: "operational",
+        }),
+        new PostcardEditable({
+          to: ADDRESSES_EDITABLE[2],
+          from: ADDRESSES_EDITABLE[0],
+          front: FILE_LOCATION_4X6,
+          back: FILE_LOCATION_4X6,
+          use_type: "operational",
+        }),
+        new PostcardEditable({
+          to: ADDRESSES_EDITABLE[4],
+          from: ADDRESSES_EDITABLE[1],
+          front: FILE_LOCATION_4X6,
+          back: FILE_LOCATION_4X6,
+          use_type: "operational",
+        }),
+        new PostcardEditable({
+          to: ADDRESSES_EDITABLE[5],
+          from: ADDRESSES_EDITABLE[0],
+          front: FILE_LOCATION_4X6,
+          back: FILE_LOCATION_4X6,
+          use_type: "operational",
+        }),
+      ];
+
+      // Create all postcards with error handling
+      for (const postcard of postcardsToCreate) {
+        try {
+          await postcardsApi.create(postcard);
+        } catch (error) {
+          console.log(`Failed to create postcard: ${error}`);
+        }
+      }
```

Comment:
There's a lot of repetition in the setup. Explore ways of parameterizing this via one of the options:

1. Create a factory function
2. Base template + variations
3. via an array mapping

```suggestion
      const basePostcard = {
        front: FILE_LOCATION_4X6,
        back: FILE_LOCATION_4X6,
        use_type: "operational" as const,
      };

      const addressPairs = [
        [1, 0],
        [3, 0],
        [6, 1],
        [2, 0],
        [4, 1],
        [5, 0],
      ];

      const postcardsToCreate = addressPairs.map(
        ([to, from]) =>
          new PostcardEditable({
            ...basePostcard,
            to: ADDRESSES_EDITABLE[to],
            from: ADDRESSES_EDITABLE[from],
          })
      );
```

Further consider using `Promise.all()` to create the postcards in parallel.